shell/imports/server/migrations.js

```diff
@@ -787,6 +787,49 @@ function setIpBlacklist(db, backend) {
   }
 }
 
+function notifyIdentityChanges(db, backend) {
+  // Notify users who might be affected by the identity model changes.
+  //
+  // Two types of users are affected:
+  // - Users who have multiple identities with differing names.
+  // - Users who have identities that are shared with other users.
+  //
+  // However, the second group seems like it must be a subset of the first group. So we only check
+  // for the first.
+
+  const names = {};
+  Meteor.users.find({ "profile.name": { $exists: true } }, { fields: { "profile.name": 1 } })
+      .forEach(user => {
```

Comment:
What about profiles for which the `name` field is not filled in yet? (See https://github.com/sandstorm-io/sandstorm/blob/v0.207/shell/packages/sandstorm-db/profile.js#L189-L222.)